README.md

```diff
@@ -25,3 +25,96 @@ either their lease duration expires or someone releases the lease with their UUI
 specifies the lease duration when they acquire it.
 
 For implementation details, see [the architecture document](doc/architecture.md)
+
+# API
+
+The server exposes a REST API to acquire and release leases for clusters. The subsections
+herein list each endpoint.
+
+## `POST /lease`
+
+Acquire a new lease.
+
+### Request Body
+
+```json
+{"max_time": "30"}
+```
+
+Note that the value of `max_time` is the maximum lease duration in seconds. After this duration
+expires, the lease will be automatically released.
+
+### Responses
+
+Unless otherwise noted, all responses except for `200 OK` indicate that the lease was not acquired.
+
+In non-200 response code scenarios, a body may be returned with an explanation of the error,
+but the existence or contents of that body are not guaranteed.
+
+#### `401 Bad Request`
+
+This response code is returned with no specific body if the request body was malformed.
+
+#### `500 Internal Server Error`
+
+This response code is returned if any of the following occur:
+
+- The server couldn't communicate with the Kubernetes Master to get the service object
+- The server couldn't communicate with the GKE API
+- The annotations on the service object were malformed
+- A cluster was available, but the new lease information couldn't be saved
+- An expired lease exists but it points to a non-existent cluster
+- The lease was succes
+
+#### `409 Conflict`
+
+This response code is returned if there are no clusters available for lease.
+
+#### `200 OK`
+
+This response code is returned along with the below response body if a lease could be successfully
```

Comment:
fixed in https://github.com/deis/k8s-claimer/pull/23/commits/ff5bb88493e9dd9137f1dc4190bc961849c3e6f4